EnglishFilter/sections/specific.txt

```diff
@@ -11081,7 +11081,7 @@ iwatchgameofthrones.net##.the-content > p > a[href^="http://bit.ly/"]
 msn.com##.todayshowcasead
 mirror.co.uk##.top-slot
 tempr.email##.TopBanner
-tempr.email,techcentral.ie##.topBanner
+ada.org,tempr.email,techcentral.ie##.topBanner
```

Comment:
This element is empty on my device

Reply:
Aren't we blocking the banners?

Reply:
> This element is empty on my device

What do you mean? This rule is required for the desktop version of the site.